### PR TITLE
Update fleets in welcome section

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/welcome/concepts.md [hunk 1]
```diff
@@ -13,25 +13,21 @@ An internet-connected, single-board computer running {{ $names.os.lower }} and p
...
 
 ### Device supervisor
 
-{{ $names.company.upper }}'s agent on the device, responsible for managing application updates and reporting device status.
+{{ $names.company.upper }}'s agent on the device, responsible for managing fleet updates and reporting device status.
 
 ### {{ $names.os.upper }} image
 
-A versioned build of the host OS, specific to an application and device type.
+A versioned build of the host OS, specific to an fleet and device type.
```
Comment (chrisys) [inline pages/learn/welcome/concepts.md:30]:
```suggestion
A versioned build of the host OS, specific to a fleet and device type.
```

File: pages/learn/welcome/concepts.md [hunk 2]
```diff
@@ -13,25 +13,21 @@ An internet-connected, single-board computer running {{ $names.os.lower }} and p
...
-
 ### Fleet
 
-The set of devices belonging to an application.
+A set of releases and a set of devices, encompassing the code and history of the code that applies to a fleet.
 
 ### Service
 
-An independently deployable component of an application, packaged into a Docker image and run as a container.
+An independently deployable component of a fleet, packaged into a Docker image and run as a container.
```
Comment (chrisys) [inline pages/learn/welcome/concepts.md:22]:
```suggestion
A component of a release, packaged into a Docker image and run as a container.
```

File: pages/learn/welcome/concepts.md [hunk 3]
```diff
@@ -13,25 +13,21 @@ An internet-connected, single-board computer running {{ $names.os.lower }} and p
...
 A snapshot of code collected into one or more Docker images.
 
-### Application
-
-A set of releases and a set of devices, encompassing the code and history of the code that applies to a fleet.
-
 ### Fleet
 
-The set of devices belonging to an application.
+A set of releases and a set of devices, encompassing the code and history of the code that applies to a fleet.
```
Comment (chrisys) [inline pages/learn/welcome/concepts.md:18]:
```suggestion
A group of devices sharing the same releases, variables, configuration, settings and members.
```

File: pages/learn/welcome/introduction.md [hunk 4]
```diff
@@ -8,17 +8,17 @@ title: Welcome
 
 Our [{{ $names.company.lower }} primer][primer] covers the platform from end to end, walking through all the pieces and how they fit together. We've also defined some [core concepts][concepts] you'll see throughout the documentation.
 
-Ready to dive in? The [getting started guides][getting-started] take you step-by-step through provisioning your first device and deploying your first application.
+Ready to dive in? The [getting started guides][getting-started] take you step-by-step through provisioning your first device and deploying your first fleet.
```
Comment (chrisys) [inline pages/learn/welcome/introduction.md:11]:
```suggestion
Ready to dive in? The [getting started guides][getting-started] take you step-by-step through creating your first fleet, provisioning your first device and deploying your first release.
```

File: pages/learn/welcome/introduction.md [hunk 5]
```diff
@@ -8,17 +8,17 @@ title: Welcome
...
 
-Once you're up on the basics, we've got all the info you need to build even the most advanced applications and manage larger fleets:
+Once you're up on the basics, we've got all the info you need to build and manage advanced larger fleets:
```
Comment (chrisys) [inline pages/learn/welcome/introduction.md:13]:
```suggestion
Once you're up on the basics, we've got all the info you need to build and manage larger fleets with more advanced functionality:
```

File: pages/learn/welcome/introduction.md [hunk 6]
```diff
@@ -8,17 +8,17 @@ title: Welcome
...
 
 ### [Develop][develop]
 
-As you develop your application, these guides will help you configure and package your services, ensuring you have the proper container setup and that you can interact with hardware and system resources.
+As you develop your fleet, these guides will help you configure and package your services, ensuring you have the proper container setup and that you can interact with hardware and system resources.
```
Comment (chrisys) [inline pages/learn/welcome/introduction.md:17]:
```suggestion
As you develop your application and grow your fleet, these guides will help you configure and package your services, ensuring you have the proper container setup and that you can interact with hardware and system resources.
```

I think this is one of those ones where it makes sense to keep it

File: pages/learn/welcome/primer.md [hunk 7]
```diff
@@ -23,9 +23,9 @@ It is important to note that all {{ $names.company.lower }} devices, both those
...
+First, the device connects to the network and performs its early provisioning, which registers it on your dashboard. Then, the container partition is decompressed, and the device supervisor starts. This is the part that takes the most time. As soon as the supervisor starts, it registers onto the VPN and receives its unique {{ $names.company.lower }} API key. At that point, you will see the device as online in your dashboard and can use the device as normal. If the fleet that the device provisions has already had code pushed to it, the new device downloads the latest version and begins operating as expected.
```
Comment (chrisys) [inline pages/learn/welcome/primer.md:28]:
```suggestion
First, the device connects to the network and performs its early provisioning, which registers it on your dashboard. Then, the container partition is decompressed, and the device supervisor starts. This is the part that takes the most time. As soon as the supervisor starts, it registers onto the VPN and receives its unique {{ $names.company.lower }} API key. At that point, you will see the device as online in your dashboard and can use the device as normal. If the fleet that the device joins already has releases, the new device downloads the latest one and begins operating.
```

File: pages/learn/welcome/primer.md [hunk 8]
```diff
@@ -35,9 +35,9 @@ First, the device connects to the network and performs its early provisioning, w
...
 
-For applications with [multiple containers][multicontainer], a `docker-compose.yml` file will need to be included at the root of your project. This configuration file specifies the services that make up your application, as well as the system resources each service has access to. Applications with a single container will have a default `docker-compose.yml` file generated if none is included.
+For fleets with [multiple containers][multicontainer], a `docker-compose.yml` file will need to be included at the root of your project. This configuration file specifies the services that make up your fleet, as well as the system resources each service has access to. Fleets with a single container will have a default `docker-compose.yml` file generated if none is included.
```
Comment (chrisys) [inline pages/learn/welcome/primer.md:40]:
```suggestion
For releases with [multiple containers][multicontainer], a `docker-compose.yml` file will need to be included at the root of your project. This configuration file specifies the services that make up your release, as well as the system resources each service has access to. Release with a single container will have a default `docker-compose.yml` file generated if none is included.
```

File: pages/learn/welcome/production-plan.md [hunk 9]
```diff
@@ -90,7 +90,7 @@ If your application generates enough log data that you need an analytics platfor
 
 ### Update testing
 
-Make sure to test all the update functionality you can before your devices are out in the wild and inaccessible. For both application and host OS updates, {{ $names.company.lower }} includes a number of features to make sure the update process doesn't affect your ability to connect to your devices and restore functionality, but you'll still want to understand the factors that can affect the update process.
+Make sure to test all the update functionality you can before your devices are out in the wild and inaccessible. For both the application and host OS updates, {{ $names.company.lower }} includes a number of features to make sure the update process doesn't affect your ability to connect to your devices and restore functionality, but you'll still want to understand the factors that can affect the update process.
```
Comment (chrisys) [inline pages/learn/welcome/production-plan.md:93]:
```suggestion
Make sure to test all the update functionality you can before your devices are out in the wild and inaccessible. For both release and host OS updates, {{ $names.company.lower }} includes a number of features to make sure the update process doesn't affect your ability to connect to your devices and restore functionality, but you'll still want to understand the factors that can affect the update process.
```

File: pages/learn/welcome/introduction.md (hunk 6 above)
Comment (vipulgupta2048) [reply pages/learn/welcome/introduction.md:17]: Agreed

**Comment by vipulgupta2048** [comment]
ready to go, rebase when the turn comes for it.

File: pages/learn/welcome/primer.md [hunk 10]
```diff
@@ -37,7 +37,7 @@ First, the device connects to the network and performs its early provisioning, w
...
 
-For fleets with [multiple containers][multicontainer], a `docker-compose.yml` file will need to be included at the root of your project. This configuration file specifies the services that make up your fleet, as well as the system resources each service has access to. Fleets with a single container will have a default `docker-compose.yml` file generated if none is included.
+For releases with [multiple containers][multicontainer], a `docker-compose.yml` file will need to be included at the root of your project. This configuration file specifies the services that make up your release, as well as the system resources each service has access to. Release with a single container will have a default `docker-compose.yml` file generated if none is included.
```
Comment (chrisys) [inline pages/learn/welcome/primer.md:40]:
```suggestion
For releases with [multiple containers][multicontainer], a `docker-compose.yml` file will need to be included at the root of your project. This configuration file specifies the services that make up your release, as well as the system resources each service has access to. Releases with a single container will have a default `docker-compose.yml` file generated if none is included.
```

I missed this one before - sorry!

**Review by chrisys — APPROVED** [review]

**Comment by chrisys** [comment]
@balena-ci rebase

**Pull request merged**